src/ultis/YupValidation.tsx

```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup';
```

Comment:
yarn add @types/yup

Reply:
bro, it's already add
When I change to `import * as Yup from '@type/yup' 
It shows error